build_test/test/multi_asset_reader_test.dart

```diff
@@ -60,5 +60,26 @@ void main() {
       ]);
       expect(assetReader.findAssets(new Glob('lib/*.dart')), [idA, idB]);
     });
+
+    test('should support the `package` arg in `findAssets`', () {
```

Comment:
Might not hurt to add a test for when no root package is defined nor is a package passed to the reader.

Reply:
Looks like with the recent push this scenario has been covered.

Reply:
this class doesn't directly throw but I added a test that it propagates errors from the wrapped readers (and used this scenario as an example)